docs/content/en/docs/Reference/auroraboot.md

```diff
@@ -0,0 +1,458 @@
+    {{% alert title="Warning" %}}
+    Make sure you have KVM enabled, this will improve the performance of your VM significantly!
+    {{% /alert %}}
+
+    This would be the way to start it via the command line, but you can also use the GUI
+
+    {{< highlight bash >}}
+      virt-install --name my-first-kairos-vm \
+                  --vcpus 1 \
+                  --memory 1024 \
+                  --cdrom build/kairos.iso.custom.iso \
+                  --disk size=30 \
+                  --os-variant opensuse-factory \
+                  --virt-type kvm
+
+    {{< / highlight >}}
+    Immediately after open a viewer so you can interact with the boot menu:
+    {{< highlight bash >}}
+    virt-viewer my-first-kairos-vm
+    {{< / highlight >}}
+
+  {{% /tab %}}
+{{< /tabpane >}}
+
+## Configuration
+
+The `AuroraBoot` configuration file reference is the following:
+
+```yaml
+# Corresponding artifact versions from the kairos release page (e.g. kubernetes version included)
+artifact_version: "v..."
+# Version of the release in github
+release_version: "v1.5.0"
+
+# Flavor
+flavor: "rockylinux"
+
+# Github repository
+repository: "kairos-io/kairos"
+
+# Container image (takes over)
+container_image: "..."
+
+# Disable netboot
+disable_netboot: true
+
+# Disable http server for serving offline generated ISOs
+disable_http_server: true
+
+# Specify a directory that will be used by auroraboot to download artifacts
+# Reuse the same to cache artifacts.
+state_dir: "/tmp/auroraboot"
+
+# Default http binding port for offline ISO generation
+listen_addr: ":8080"
+
+# Cloud config to use when booting the machine.
+cloud_config: |
+```
+
+| Option | Description |
+| ------ | ----------- |
+| `artifact_version` | Corresponding artifact versions from the kairos release page (e.g. kubernetes version included). |
+| `release_version` | Version of the release in GitHub. |
+| `flavor` | The Kairos flavor to use. See [the Kairos support matrix](/docs/reference/image_matrix/) for a list. |
+| `repository` | Github repository to use. This can either be `kairos-io/kairos` or `kairos-io/provider-kairos` for images with `k3s`. |
+| `container_image` | Container image. If prefixed with `docker://` it will try to pull from the local docker daemon. If a `container_image` is specified, `artifact_version` `flavor` and `release_version` are ignored. |
+| `disable_netboot` | Disable netboot. |
+| `disable_http_server` | Disable http server for serving offline generated ISOs. |
+| `netboot_http_port` | Specify a netboot HTTP port (defaults to `8090`). |
+| `state_dir` | Specify a directory that will be used by auroraboot to download artifacts and reuse the same to cache artifacts. |
+| `listen_addr` | Default http binding port for offline ISO generation. |
+| `cloud_config` | Inline cloud config to use for the machines. |
+
+
+To use the configuration file with `AuroraBoot`, run `auroraboot` specifying the file or URL of the config as first argument:
```

Comment:
IMO, it's better to keep the use of back ticks for commands `auroraboot` and switch all cases of `AuroraBoot` to simply be AuroraBoot or **AuroraBoot**. Not a hard request of course.

```suggestion
To use the configuration file with AuroraBoot, run `auroraboot` specifying the file or URL of the config as first argument:
```